### DISCUSSION TITLE
How to remove repository from top repositories in GitHub main page?

### DISCUSSION THREAD

**Discussion by azizsayfiddin** [community/community#43713] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

How to delete this section ?
![image](https://user-images.githubusercontent.com/114720383/211286907-78cf3b4e-7f4a-4969-b618-a9cf76dd5b0c.png)

**Comment by shree675** [comment]
@aljustiet I may be wrong, but I think it is not possible to remove that section. However, it is private to you. Only you can see that section, others cannot view your top repositories.

**Comment by slippery-seabass** [comment]
@aljustiet: Afaik, the order of 'top repositories' is calculated off of the amount of lines you've added/changed.
